po/ro.po

```diff
@@ -0,0 +1,136 @@
+# English (United Kingdom) translation for mines
```

Comment:
Ohh sorry ... my bad. Tonight I will change this in my project also and I will also add Romanian diacritics ... I was afraid that the font you used does not support them so I didn't used any but when I did a test characters like: ăîșțâ and their capital versions ĂȘÎȚÂ were correctly displayed so ... I want to get as best translations as possible.